credentialz/credentialz.proto

```diff
@@ -647,6 +635,13 @@ message PublicKey {
   string description = 3;
 }
 
+enum AuthenticationType {
+  AUTHENTICATION_UNSPECIFIED = 0;
```

Comment:
I believe it should be `AUTHENTICATION_TYPE_UNSPECIFIED`. Otherwise the linter will complain.